src/gui_impl/dialogs/event_types_selection_dialog.cpp

```diff
@@ -48,8 +48,8 @@ void EventTypesSelectionDialog::buildTree (bool only_existing_events)
     ui_.tree_widget_->setColumnWidth(ID_COLUMN_INDEX_, 0);
 
     ui_.tree_widget_->header()->setSectionResizeMode (QHeaderView::Interactive);
-    ui_.tree_widget_->header()->resizeSection (NAME_COLUMN_INDEX_, 380);
-    ui_.tree_widget_->header()->resizeSection (COLOR_COLUMN_INDEX_, 120);
+    ui_.tree_widget_->header()->resizeSection (NAME_COLUMN_INDEX_, width() / 1.65);
+    ui_.tree_widget_->header()->resizeSection (COLOR_COLUMN_INDEX_, width() / 5.5);
```

Comment:
@cbrnr They are quite different, because some event text is quite long.

![image](https://cloud.githubusercontent.com/assets/20526646/23056377/27dd56ea-f4b9-11e6-96af-eb7759ccf8d1.png)
![image](https://cloud.githubusercontent.com/assets/20526646/23056409/42528fd6-f4b9-11e6-8fef-ee3b2e924ca2.png)